docs/tutorials/azure-private-dns.md

```diff
@@ -66,7 +66,7 @@ More powerful role-assignments like `owner` or assignments on subscription-level
 
 Start off by **creating the service principal** without role-assignments.
 ```
-$ az ad sp create-for-rbac -n externaldns-sp
+$ az ad sp create-for-rbac --skip-assignment -n http://externaldns-sp
```

Comment:
the docs above say skip assignment, but the cli call was missing it so it was assigning contributor at the subscription

Reply:
Thx!
You went through it very carefully.